cob_openni2_tracker/launch/create_nite2_link.bash

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
```

Comment:
yes, the `libnite2` package no copies both headers and libs into both devel and install space and exports according cmake extras
no need to go through symlink in `~/.ros`